src/components/ScreenWrapper/BaseScreenWrapper.js

```diff
@@ -19,6 +20,8 @@ import ONYXKEYS from '../../ONYXKEYS';
 import {withNetwork} from '../OnyxProvider';
 import {propTypes, defaultProps} from './propTypes';
 
+propTypes.keyboardAvoidingViewBehavior = PropTypes.oneOf(['padding', 'height', 'position', '']).isRequired;
```

Comment:
Why not add these to `propTypes`?

Reply:
sorry why does it need to be required?

Reply:
I see what you were suggesting. Done!

Reply:
we are still passing a default in the `defaultProps` for this. so I think we can just remove this and if we want to be able to pass a string then add it here it's ok

https://github.com/Expensify/App/blob/e6b6dd2087a2b8f8dffe0b121194d733348f6cb0/src/components/ScreenWrapper/propTypes.js#L24-L25

?

Reply:
Think we still wanted to add the `''` here but maybe it's fine it will just do a warning 🤷